crud.py

```diff
@@ -1,18 +1,149 @@
-from db import async_session, engine
+"""
+crud.py
```

Comment:
There is a [convention](https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5) for REST API endpoints that we could utilize to make our endpoints more user frienty



HTTP GET /tasks
Will return all tasks

HTTP GET /tasks/<task_id>
Will return a single task

HTTP POST /tasks
Will create a new task

HTTP PUT or PATCH /tasks/<task_id>
Will update an existing task

HTTP PATCH /tasks/<task_id>/finish
Will finisjh a specified task

Also, I suggest you create a [FastAPI router](https://fastapi.tiangolo.com/reference/apirouter/) for /tasks and place all tasks-related endpoints there. This will allow you to organize your code better and avoid needing to type the ../tasks/.. prefix for your endpoints.



Also you could check out our Alma BuildSystem [API endoints](https://build.almalinux.org/docs#/) maybe you find something  useful for your app